habit/src/styles/login.js

```diff
@@ -0,0 +1,11 @@
+const React = require('react-native');
+
+const { StyleSheet } = React;
+
+module.exports = StyleSheet.create({
+    loginContainer: {
```

Comment:
We have such style in styles/common.js
Can we just use that instead. Since it is almost the same for all pages. 

It's something like pageView in common.js

Reply:
you've commented that you want to move the styles from common.js to login.js in common.js file
so, I just moved